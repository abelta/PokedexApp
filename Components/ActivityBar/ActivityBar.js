import React from 'react';
import PropTypes from 'prop-types';
import { ProgressBarAndroid, Platform, View } from 'react-native';

const ActivityBar = props => (
  <View style={{ height: 2 }}>
    {
      props.active && Platform.OS === 'android' &&
      <ProgressBarAndroid color="grey" styleAttr="Horizontal" style={{ marginTop: -7, marginBottom: -6 }} {...props} />
    }
  </View>
);

ActivityBar.propTypes = {
  active: PropTypes.bool,
};

ActivityBar.defaultProps = {
  active: false,
};

export default ActivityBar;
