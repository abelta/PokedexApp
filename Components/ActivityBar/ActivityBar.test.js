import React from 'react';
import renderer from 'react-test-renderer';
import ActivityBar from '.';

describe('ActivityBar', () => {
  it('renders correctly', () => {
    const tree = renderer.create(<ActivityBar />).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
