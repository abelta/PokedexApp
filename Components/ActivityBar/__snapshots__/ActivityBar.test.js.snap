// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ActivityBar renders correctly 1`] = `
<View
  style={
    Object {
      "height": 2,
    }
  }
/>
`;
